charts/ccsm-helm/charts/zeebe/templates/service.yaml

```diff
@@ -19,5 +19,13 @@ spec:
     - port: {{ .Values.service.commandPort }}
       protocol: TCP
       name: {{ default "command" .Values.service.commandName }}
+    {{- if hasKey .Values.service "extraPorts" }}
```

Comment:
hasKey is not necessary as far as I know. You can just do `{{- if .Values.service.extraPorts }}`
see https://helm.sh/docs/chart_template_guide/control_structures/#ifelse

Reply:
@Zelldon  You are right. I like to have the has key in case there is some individual runs the helm chart with his own values.yaml. If that is the case, (someone runs the chart without the extra ports key), then the chart will break and will not install 
I can remove it if you prefer :)

Reply:
> If that is the case, (someone runs the chart without the extra ports key), then the chart will break and will not install

Not sure whether I get right what you mean, but the `extraPorts` are per default `[]` which means the condition will be false, since:

> A pipeline is evaluated as false if the value is:
> 
> a boolean false
> a numeric zero
> an empty string
> a nil (empty or null)
> an empty collection (map, slice, tuple, dict, array)

Reply:
You are right. But if somebody accidentally omits the `extraPorts []` then it will break 

But yeah they shouldn't never run into this since we are setting it as a default (empty)